### PR TITLE
CH166 integrate pattern library

### PR DESCRIPTION
**Pull request by andymantell**
This should be reviewed in conjunction with the following:
https://github.com/LandRegistry/digital-register-acceptance-tests/pull/128
https://github.com/LandRegistry/centos-dev-env/pull/90

Note: After you have switched to this branch you will find that `service/static/govuk_frontend_toolkit/` is shown as untracked. This is the remnants of a git submodule which no longer exists - please delete it

### CODE REVIEW

File: service/__init__.py [hunk 1]
```diff
@@ -38,7 +38,14 @@ def before_first_request():
     Set price only after all services are running - digital-register-api in particular.
     """
 
-    price = "{0:.2f}".format(api_client.get_pound_price())
+    price = api_client.get_pound_price()
+
+    # If the price is a round number of pounds we don't want to display any decimal places
+    if price.is_integer():
```
Comment (russwillis) [inline service/__init__.py:44]: is this an arbitrary  or a UR one? :)

Comment (andymantell) [inline service/__init__.py:44]: @russwillis digital-register-frontend repeatedly deviated from the prototype in this respect, so I fixed it... So, I would say this is UR driven :-)

File: service/static/.land-registry-elements/assets/javascripts/govuk-template.js [hunk 2]
```diff
@@ -77,20 +77,6 @@
 (function() {
   "use strict"
 
-  // fix for printing bug in Windows Safari
```
Comment (russwillis) [inline service/static/.land-registry-elements/assets/javascripts/govuk-template.js:80]: is this not a bug anymore?

Comment (andymantell) [inline service/static/.land-registry-elements/assets/javascripts/govuk-template.js:80]: @russwillis This has come from upstream in the gov uk template. See this commit: https://github.com/alphagov/govuk_template/commit/4348f4319396369b65015544c5d41bf3ca704067

**Pull request merged**